frontend/src/app/main/ui/dashboard/placeholder.cljs

```diff
@@ -11,20 +11,18 @@
    [rumext.alpha :as mf]))
 
 (mf/defc empty-placeholder
-  [{:keys [dragging? default?] :as props}]
+  [{:keys [dragging? on-create-clicked project] :as props}]
   (cond
     (true? dragging?)
     [:div.grid-row.no-wrap
      [:div.grid-item]]
 
-    (true? default?)
-    [:div.grid-empty-placeholder.drafts {:data-test "empty-placeholder"}
-     [:div.text
-      [:& i18n/tr-html {:label "dashboard.empty-placeholder-drafts"}]]]
-
     :else
     [:div.grid-empty-placeholder
-     [:img.ph-files {:src "images/ph-file.svg"}]]))
+     [:button.create-new {:on-click (partial on-create-clicked project)}
+      (tr "dashboard.new-file")
+      ]
+     ]))
```

Comment:
I think it can be shorter with something lile:
      (tr "dashboard.new-file")]]))

Reply:
true¡¡